tests/save.rs

```diff
@@ -62,8 +62,8 @@ fn save_fancy() {
     assert_eq!(pre_layer.iter().count(), post_layer.iter().count());
 
     for glyph in pre_layer.iter() {
-        let other = post_layer.get_glyph(&glyph.name);
-        assert!(other.is_some(), "missing {}", &glyph.name);
+        let other = post_layer.get_glyph(&glyph.name());
+        assert!(other.is_some(), "missing {}", &glyph.name());
```

Comment:
If we're already returning a reference from the method, we won't need to take a reference here:

```suggestion
        let other = post_layer.get_glyph(glyph.name());
        assert!(other.is_some(), "missing {}", glyph.name());
```